### PR TITLE
nexx360 Bid Adapter: new functionalities and endpoint update

### PR DESCRIPTION
**Pull request by gchicoye**
## Type of change

- [ ] Bugfix
- [X] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

We reworked on the bid adapter to added multiple features such as:
- Prebid 7.0 Floor module support
- Multi format bids

We've also updated the endpoint to be compliant with new data structure.

## Other information

We've submitted a linked PR on prebid.github.io, with updated documentation.
https://github.com/prebid/prebid.github.io/pull/4140

### CODE REVIEW

**Review by Fawke — COMMENTED** [review]
Hi @gchicoye,

The changes look good. Before I merge this, can you please improve your test coverage to make it at least 80%?

You can check your current test coverage by running the commands, `gulp test-coverage` followed by `gulp view-coverage`.

File: modules/nexx360BidAdapter.js [hunk 1]
```diff
@@ -1,176 +1,333 @@
...
+
+//     enrichImp(imp, bid, floor);
+//     return imp;
+//   });
+//   return {
+//     method: 'POST',
+//     url: REQUEST_URL,
+//     data: data
+//   }
+// }
```
Comment (Fawke) [inline modules/nexx360BidAdapter.js:200]: Do we need to keep this commented function?

Comment (gchicoye) [reply modules/nexx360BidAdapter.js]: Indeed, removed!

**Comment by gchicoye** [comment]
Test coverage above 80% should be fixed by now :)

**Comment by gchicoye** [comment]
hi @Fawke, is it OK for you now?

**Review by Fawke — APPROVED** [review]
LGTM

**Pull request merged**